queries.sql

```diff
@@ -12,4 +12,55 @@ SELECT * FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name != 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;   // const user = userData ? JSON.parse(userData) : null
+
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(100);
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
👏 Great job on implementing the first three parts of the transaction! 👏 However, according to the project requirements, there is one more step needed after the rollback. Let's address this to ensure completion:

- [ ] **Verification After Rollback:** After rolling back the transaction, it's essential to verify that the `species` column has returned to its state before the transaction. Please add this fourth step to complete the transaction process.
- [ ] Additionally, please consider to add the screenshot in the PR description too.

By incorporating this verification step, your implementation will align perfectly with the project requirements. Keep up the excellent work! 🚀🔍